src/Temporalio/Client/TemporalClient.cs

```diff
@@ -112,5 +143,33 @@ internal partial class Impl : Interceptors.ClientOutboundInterceptor
             /// </summary>
             internal TemporalClient Client { get; init; }
         }
+
+#pragma warning disable CA1822 // We don't want to force plugin methods to be static
+        /// <summary>
+        /// Placeholder.
+        /// </summary>
+        internal class RootPlugin : Plugin
```

Comment:
Would recommend making this implement the interface w/ a throw `NotImplementedException` for init